### ISSUE TITLE
Command - shorther version for project_generator

### ISSUE THREAD

**Issue by 0xc0170** [0xc0170/project_generator#74]
I hate writing project_generator all the time, too long :neckbeard:... there are couple of options we have

```
pg
pgen
prg
prgen
```

pg is a module already in the pypi, shall we aim for another?

**Comment by matthewelse** [comment]
I was starting to get fed up with this too :+1: the only other potential conflict I can see is PostgreSQL but since the kind of engineers using that are unlikely to be using project_generator too I think we'll be fine.

I think for backwards compatibility we should keep the project_generator command line name and just add another one

**Comment by matthewelse** [comment]
+1 for pgen

**Comment by 0xc0170** [comment]
Yes, ah project_generator stays. Just need add a new to the console . 

For me +1 to pgen and prgen

**Comment by matthewelse** [comment]
```python
'console_scripts': [
            "project_generator=project_generator.main:main",
            "pgen=project_generator.main:main"
        ]
```

**Comment by 0xc0170** [comment]
pgen is available, resolved via #77
